backend/api/index.py

```diff
@@ -29,10 +29,13 @@ async def lifespan(app: FastAPI):
 @app.post("/download")
 async def download_data():
     try:
+        print("==> Download triggered")
         downloader = Downloader()
         downloader.run()
```

Comment:
I'd suggest using `downloader.download_tomorrow()`, which makes it easier to comprehend the code/this endpoint. 😄
```suggestion
        downloader.download_tomorrow()
```